assets/js/src/newsletter_editor/behaviors/TextEditorBehavior.js

```diff
@@ -21,6 +22,14 @@ define([
       plugins: 'link textcolor colorpicker mailpoet_shortcodes',
       configurationFilter: function configurationFilter(originalConfig) { return originalConfig; }
     },
+    initialize: function initialize() {
+      var that = this;
+      App.getChannel().on('dragStart', function dragStart() {
```

Comment:
This mostly looks good, but I think not cleaning up the callback upon destruction of the `TestEditorBehavior` instance causes a memory leak and an ever growing number of listeners.

You can test this by adding a text block, editing its text, deleting it and executing this command in console:
```
EditorApplication.getChannel()._events.dragStart.length
```
If you repeat the same steps, you would expect the number of listeners to keep growing, because old ones are not being cleaned up.

Reply:
Good catch! I've fixed it using `listenTo`